Tests/CwlViewsCatalog_iOS/Sources/PickerView.swift

```diff
@@ -44,27 +35,23 @@ func pickerView(_ viewState: PickerViewState, _ navigationItem: NavigationItem)
 				marginEdges: .allLayout,
 				.view(
 					Label(
-						.text <-- viewState.selectedRow.map { value in return "\(value)"},
+						.text <-- viewState.selectedRow.map { value in return "\(value.data)"},
 						.font -- UIFont.monospacedDigitSystemFont(ofSize: 17, weight: .regular)
 					)
 				),
 				.space(),
 				.view(
 					PickerView<String>(
 						.backgroundColor -- .red,
-						.pickerData -- .reload([PickerComponent<String>(elements: [.text("1"), .text("2"), .text("3"), .view("4")]), PickerComponent<String>(elements: [.text("5"), .text("6"), .text("7"), .view("🏁")])]),
+						.pickerData -- .reload([PickerComponent<String>(elements: ["1", "2", "3", "4"]), PickerComponent<String>(elements: ["1", "2", "3", "4"])]),
 						.rowSelected --> viewState.selectedRow,
-						.viewConstructor -- { data in
-							Label(.text <-- data)
-						}
+						.title -- { $0.data }
+//						.viewConstructor -- { data in
```

Comment:
I have tested this and it works, so I left it in as a comment for easy testing purposes so one can see that `.title` also works as expected and when both are set `viewConstructor` is given precedence.